packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -160,6 +169,16 @@ export default compose( [
 		const icon = blockType.icon;
 		const getAccessibilityLabelExtra = blockType.__experimentalGetAccessibilityLabel;
 
+		const rootBlockId = getBlockHierarchyRootClientId( clientId );
+		const rootBlock = getBlock( rootBlockId );
+		const parentBlockId = getBlock( clientId ).parent;
+		const parentBlock = getBlock( parentBlockId );
+		const isGroupRoot = rootBlock.name === 'core/group';
+		const hasRootInnerBlocks = rootBlock.innerBlocks.length !== 0;
+		const isMediaTextParent = parentBlock && parentBlock.name === 'core/media-text';
```

Comment:
We don’t have to exclude Media & Text block. We want the same UX everywhere InnerBlocks are used

Reply:
Makes sense. Do we want to display toolbar above each `Media & Text` children when `Media & Text` is nested in `Group` component or only over the whole `Media & Text` component?

Reply:
I've attached a gif presenting how `FloatingToolbar` behaves once I remove Media&Text exclusion
 <img src="https://user-images.githubusercontent.com/22746080/64641867-55c2bd80-d40d-11e9-8bc0-47321225fa5a.gif" width="300" /> 

Please let me know if it's correct.

Reply:
Right, we need to show it whenever a block has InnerBlocks. If you check out the prototype link in the issue same thing can be seen with the column block.